org.kde.labplot.json

```diff
@@ -0,0 +1,437 @@
+{
+    "app-id": "org.kde.labplot",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.9",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.9",
+    "sdk": "org.kde.Sdk",
+    "command": "labplot",
+    "rename-icon": "labplot",
+    "finish-args": [
+        "--filesystem=host",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/lib/*.la",
+        "/lib/*.a",
+        "/mkspecs",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "gsl",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz",
+                    "sha256": "6a99eeed15632c6354895b1dd542ed5a855c0f15d9ad1326c6fe2b2c9e423190",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 1267,
+                        "stable-only": true,
+                        "url-template": "http://ftp.gnu.org/gnu/gsl/gsl-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "hdf5",
+            "buildsystem": "simple",
```

Comment:
this should not be simple, there is `subdir` to point it to a subdirectory of toplevel source directory where `configure` is present.